samples/testing-frameworks/appium/client-side/csharp/biometric-sample-test/BitbarBiometricSample/TestBiometricsIOS.cs

```diff
@@ -0,0 +1,82 @@
+﻿using OpenQA.Selenium.Appium.iOS;
+
+namespace BitbarBiometricSample
+{
+    [TestFixture]
+    public class TestBiometricsIOS
+    {
+        private readonly String SCREENSHOT_DIRECTORY = AppDomain.CurrentDomain.BaseDirectory + "/screenshots/";
+        private RemoteWebDriver driver;
+
+        [OneTimeSetUp]
+        public void InitDriver()
+        {
+            const String AppiumHubURL = "https://appium.bitbar.com/wd/hub";
+
+            AppiumOptions capabilities = new();
+            //Platfrom dependent 
+            capabilities.AddAdditionalCapability("platformName", "iOS");
+            capabilities.AddAdditionalCapability("deviceName", "iPhone device");
+            capabilities.AddAdditionalCapability("automationName", "XCUITest");
+            //Customizable
+            capabilities.AddAdditionalCapability("bitbar_project", "C# Appium Automated Test");
+            capabilities.AddAdditionalCapability("bitbar_testrun", "iOS Run");
+            capabilities.AddAdditionalCapability("bitbar_device", "Apple iPhone 11");
+            //Required
+            capabilities.AddAdditionalCapability("bitbar_apiKey", "<YOUR_BITBAR_API_KEY>");
+            capabilities.AddAdditionalCapability("bitbar_app", "<APP_ID>");
+            //Biometrics capability
+            capabilities.AddAdditionalCapability("biometricInstrumentation", true);
+
+            driver = new IOSDriver<IOSElement>(new Uri(AppiumHubURL), capabilities, TimeSpan.FromSeconds(300));
+        }
+
+        [OneTimeTearDown]
+        public void QuitDriver()
+        {
+            driver.Quit();
+        }
+
+        [Test]
+        public void Test_Biometric_Authentication()
+        {
+            Console.WriteLine("Test execution started:");
+
+            //Accept initial biometric dialog
```

Comment:
Waiting for dialog to display with `ExpectedConditions.alertIsPrestent()` seems to doesn't work properly in the newest WebDriver nuget package. Even if `alertIsPresent()` returns true `driver.SwitchTo().Alert().Accept()` returns an error saying the `iOS alert cannot be nil`.  So far didn't find any better option to make sure the test will continue only after the dialog is displayed.